common/rs-config.h

```diff
@@ -87,6 +87,105 @@ namespace rs2
 
         static config_file& instance();
 
+        // Retrieves a value from a nested JSON structure using dot notation
+        template< typename T >
+        T get_nested( const std::string & path ) const
+        {
+            std::istringstream ss( path );
+            std::string token;
+            const rsutils::json * current = &_j;
+
+            while( std::getline( ss, token, '.' ) )
+            {
+                if( ! current->contains( token ) )
+                {
+                    return config_value( "" );
+                }
+                current = &( *current )[token];  // getting to the next level in the JSON structure
+            }
+            return current->get< T >();
```

Comment:
`get` might throw if value is of wrong type.
You can use `get_ex` to get value and success indication.